### PR TITLE
Added Bower as dependency, created install-all target for npm

### PR DESCRIPTION
**Pull request by giftkugel**
Added Bower as dependency, created install-all target for npm

### CODE REVIEW

**Comment by hendrikebbers** [comment]
fix for https://github.com/canoo/dolphin-platform/issues/795

File: package.json [hunk 1]
```diff
@@ -18,7 +19,7 @@
     "watch": "./node_modules/.bin/browser-sync start --port 3000 --proxy 'localhost:9090' --no-ui --files '*.html, demo/*.html, test/*.html' --startPath '/components/dolphin-platform-polymer'"
   },
   "devDependencies": {
-    "bower": "1.8.2",
+    "bower": "^1.8.2",
```
Comment (hendrikebbers) [inline package.json]: Please use fix version ;)

**Comment by hendrikebbers** [comment]
@giftkugel Travis file must be updated.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   },
   "license": "Apache-2.0",
   "scripts": {
+    "install-all": "npm install && ./node_modules/.bin/bower install --force-latest",
```
Comment (hendrikebbers) [inline package.json]: I think we can remove `--force-latest`

**Comment by hendrikebbers** [comment]
@giftkugel should we simply close this one without merging it?

**Comment by hastebrot** [comment]
@hendrikebbers Can help you with the Travis issue. It's a problem with Chrome and the updated Travis docker image.

**Comment by giftkugel** [comment]
Closed, because already covered with [Pull Request 20](https://github.com/canoo/dolphin-platform-polymer/pull/20)

**Pull request closed without merging**